README.md

```diff
@@ -136,6 +136,16 @@ Most of the time, the reason for failure is present in the logs.
 * `:LspStop <client_id>` Defaults to stopping all buffer clients.
 * `:LspRestart <client_id>` Defaults to restarting all buffer clients.
 
+## Highlights
+
+* `LspInfoBorder` set the color of `LspInfo` window border. use
+  ```lua
+  require('lspconfig.ui.windows').default_options {
+    border = 'single' or other style
```

Comment:
I think it could be helpful to point to the docs for which values are accepted.

```suggestion
    -- Accepts the same values as the `border` option to `:h nvim_open_win()`
    border = 'single'
```